numba/parfors/array_analysis.py

```diff
@@ -31,6 +32,8 @@
 import llvmlite
 from numba.np.unsafe.ndarray import to_fixed_tuple
 
+_logger = logging.getLogger(__name__)
```

Comment:
Either we should switch everything to logger or use the existing NUMBA_DEBUG_ARRAY_OPT.  I suggest the latter because I see no compelling interest to switch at this point.